server/routes/makeAReferral/makeAReferralController.ts

```diff
@@ -496,16 +497,37 @@ export default class MakeAReferralController {
 
   async viewRiskInformation(req: Request, res: Response): Promise<void> {
     const referral = await this.interventionsService.getDraftReferral(res.locals.user.token.accessToken, req.params.id)
-    const [serviceUser, riskSummary] = await Promise.all([
-      this.communityApiService.getServiceUserByCRN(referral.serviceUser.crn),
-      this.assessRisksAndNeedsService.getRiskSummary(referral.serviceUser.crn, res.locals.user.token.accessToken),
-    ])
+    const serviceUser = await this.communityApiService.getServiceUserByCRN(referral.serviceUser.crn)
+    if (config.apis.assessRisksAndNeedsApi.riskSummaryEnabled) {
+      await this.displayOasysRiskInformationPage(res, referral, serviceUser)
+    } else {
+      await this.displayRiskInformationPage(res, referral, serviceUser)
+    }
+  }
 
+  private async displayRiskInformationPage(res: Response, referral: DraftReferral, serviceUser: DeliusServiceUser) {
+    const presenter = new RiskInformationPresenter(referral)
+    const view = new RiskInformationView(presenter)
+    ControllerUtils.renderWithLayout(res, view, serviceUser)
+  }
+
+  private async displayOasysRiskInformationPage(
+    res: Response,
+    referral: DraftReferral,
+    serviceUser: DeliusServiceUser
+  ) {
+    const { accessToken } = res.locals.user.token
+    const riskSummary = await this.assessRisksAndNeedsService.getRiskSummary(referral.serviceUser.crn, accessToken)
+    // TODO: If riskSummary === null show standard risk information flow or show empty oasys risk information?
```

Comment:
I'm not sure about this - I'd lean towards showing the standard risk information flow, but is there someone we can ask about this for a second opinion (possibly Leigh?) and capture the response in a Trello ticket so we make sure to revisit it soon?

Reply:
I've removed it now. when it is null it simply provides empty information